runtime/codert_vm/module.xml

```diff
@@ -23,6 +23,7 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 <module>
 	<artifact type="static" name="j9codert_vm">
 		<include-if condition="spec.flags.module_codert_vm" />
+		<exclude-if condition="spec.linux_riscv.*" />
```

Comment:
Do we need/want the equivalent changes for cmake?

Reply:
The whole changes are based on UMA. We are not doing cmake for the moment.